src/main/plugins/org.dita.pdf2/xsl/fo/troubleshooting-elements.xsl

```diff
@@ -0,0 +1,205 @@
+<?xml version='1.0'?>
+
+<!-- 20170503 SCH: Add support for troubleshooting elements. -->
+
+<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+    xmlns:fo="http://www.w3.org/1999/XSL/Format"
+    xmlns:dita2xslfo="http://dita-ot.sourceforge.net/ns/200910/dita2xslfo"
+    xmlns:opentopic="http://www.idiominc.com/opentopic"
+    xmlns:opentopic-index="http://www.idiominc.com/opentopic/index"
+    exclude-result-prefixes="opentopic opentopic-index dita2xslfo"
+    version="2.0">
+
+    <!-- Determines whether to generate titles for task sections. Values are YES and NO. -->
+    <xsl:param name="GENERATE-TASK-LABELS">

```

Comment:
There has been a discussion about creating specific label overrides, but I don't know where we ended up with it. Should I create a trouble label variable or have we decided on a generic one for all labels?